### PR TITLE
[discord-game-sdk] Won't compile with certain compilers

### PR DESCRIPTION
**Pull request by makinori**
When compiling Discord Game SDK with the latest version of GCC `10.1.0` (though I think this bug seems to be somewhere between `4.1.0` and `4.8.0`), it wont find `int64_t`. As far as I know, certain compilers on certain platforms don't always include the same headers by default. This should probably be fixed on Discord's side but my solution works perfectly for the program we're working on.

### CODE REVIEW

**Comment by makitsune** [comment]
Ah it seems Windows doesn't understand `-include`. I'll update right away

**Comment by BillyONeal** [comment]
The right way to fix this is to author a patch to include the correct headers. And ideally submit that patch upstream so that they fix it there

**Comment by makitsune** [comment]
@BillyONeal Oh! Of course, thank you. I'm still new to vcpkg haha

**Comment by BillyONeal** [comment]
You're welcome, and thanks for your contribution! :)

**Comment by sylveon** [comment]
The right place to fill an issue would be https://github.com/discord/gamesdk-and-dispatch/issues.

The code isn't open for PRs however so we have to rely on the people at Discord to fix it.

File: ports/discord-game-sdk/portfile.cmake [hunk 1]
```diff
@@ -15,6 +15,7 @@ vcpkg_extract_source_archive_ex(
     OUT_SOURCE_PATH SOURCE_PATH
     ARCHIVE ${ARCHIVE}
     NO_REMOVE_ONE_LEVEL
+    PATCHES "include-cstdint.patch" # allows compiling on newer versions of GCC
```
Comment (NancyLi1013) [inline ports/discord-game-sdk/portfile.cmake:18]:
```suggestion
    PATCHES 
        include-cstdint.patch # allows compiling on newer versions of GCC 
```

**Review by NancyLi1013 — COMMENTED** [review]
Could you please also bump the version as` 2.5.6-1` in `CONTROL `file?

File: ports/discord-game-sdk/portfile.cmake (hunk 1 above)
Comment (NancyLi1013) [reply ports/discord-game-sdk/portfile.cmake:18]: Could you please also update this?

**Review by NancyLi1013 — APPROVED** [review]

**Comment by dan-shaw** [comment]
Thanks for the PR!

**Comment by sylveon** [comment]
@makitsune I'm looking at this to fill an issue on Discord's repo and I'm a bit confused... `types.h` includes `ffi.h`, which includes `stdint.h`, shouldn't that take care of it?

**Pull request merged**